app/views/finders/show.atom.builder

```diff
@@ -0,0 +1,21 @@
+atom_feed do |feed|
+  feed.title(@finder.name)
+
+  dates = []
+  @results.results.map(&:metadata).each do |r|
```

Comment:
Actually, after chatting to @edds, he suggested the results are already in date order, and could just pick the date of the first? Using the same date field that rummager is ordered by.